fedmsg_meta_umb/ci.py

```diff
@@ -39,16 +39,22 @@ def subtitle(self, msg, **config):
                         msg['topic'].split('.')[-1])
             except KeyError:
```

Comment:
As both return None, could caught in one line with:

except (KeyError, TypeError):

Reply:
+1